services/WeChat.json

```
@@ -3,6 +3,10 @@
     "documents": {
         "tos": {
             "fetch": "https://www.wechat.com/en/service_terms.html",
+            "remove": [
+                "#wechat-terms-of-service-usa-specific-terms-",
+                "#wechat-terms-of-service-australia-specific-terms-"
+            ],
```

```suggestion
            "remove": {
                "startBefore": "#wechat-terms-of-service-usa-specific-terms-",
                "endBefore": "#wechat-terms-of-service-european-union-specific-terms-"
            },
```
This way not only titles but also contents are removed ! :smiley:

---

Of course! 🤦‍♀️ Thanks!